src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -4649,6 +4644,14 @@ private bool IsUsingAliasInScope(string name)
             }
         }
 
+        private static void WarnOnAccessOfOffDefault(CSharpSyntaxNode node, BoundExpression boundLeft, DiagnosticBag diagnostics)
+        {
+            if (boundLeft != null && boundLeft.Kind == BoundKind.DefaultOperator && boundLeft.ConstantValue == ConstantValue.Null)
+            {
```

Comment:
interesting .why not just check for constant null. i guess this is just to match old behavior.